app/celery/tasks.py

```diff
@@ -93,7 +92,6 @@
     send_notification_to_queue,
 )
 from app.notifications.validators import check_service_over_daily_message_limit
-from app.service.utils import service_allowed_to_send_to
```

Comment:
Are we still checking this somewhere in the flow? Ie making sure that a trial service isn't sending to recipients it's not allowed to.

Reply:
yup we moved this https://github.com/cds-snc/notification-api/pull/1502/files